snowfakery/parse_recipe_yaml.py

```diff
@@ -146,14 +146,34 @@ def removeline_numbers(dct: Dict) -> Dict:
     return {i: dct[i] for i in dct if i != "__line__"}
 
 
+def _coerce_to_string(val, context):
+    print(val, type(val))
```

Comment:
Was this print statement intentionally left in?

Reply:
Oops, no, fixed in a0fb7ef.